azkaban-common/src/main/java/azkaban/project/JdbcProjectImpl.java

```diff
@@ -905,7 +905,7 @@ public Props fetchProjectProperty(final int projectId, final int projectVer,
                   propsName);
 
       if (properties == null || properties.isEmpty()) {
-        logger.warn("Project " + projectId + " version " + projectVer + " property " + propsName
+        logger.debug("Project " + projectId + " version " + projectVer + " property " + propsName
```

Comment:
it's not just fetching override properties here, right?

Reply:
Correct